sql/field.h

```diff
@@ -3764,6 +3771,7 @@ class Column_definition: public Sql_alloc
   uint32 srid;
   Field::geometry_type geom_type;
   engine_option_value *option_list;
+  enum_column_versioning versioning;
```

Comment:
Guide forces us to use prefix enum_ https://dev.mysql.com/doc/internals/en/naming-conventions.html